kernel/kernel-defaults/src/test/scala/io/delta/kernel/defaults/ChecksumSimpleComparisonSuite.scala

```diff
@@ -65,11 +70,13 @@ class ChecksumSimpleComparisonSuite extends DeltaTableWriteSuiteBase with TestUt
         .forEach(hook => hook.threadSafeInvoke(engine))
       spark.sql(s"CREATE OR REPLACE TABLE delta.`${sparkTablePath}` (id LONG) USING DELTA")
       assertChecksumEquals(engine, sparkTablePath, kernelTablePath, 0)
+      maybeDeleteChecksum(kernelTablePath, 0)
```

Comment:
it's not clear to me why we delete it here? should it actually be before each commit (except the first one)?

Reply:
actually should it be part of `commitSparkChangeToKernel`? removes the version prior?